cl/search/management/commands/cl_update_index.py

```diff
@@ -154,6 +159,10 @@ def handle(self, *args, **options):
                 sys.exit(1)
             elif options.get('items'):
                 self.add_or_update(*options['items'])
+            elif options.get('source'):
+                source_val = options['source']
+                if source_val == Docket.RECAP:
+                    self.add_or_update_all_recap_files()
```

Comment:
I think you have a misunderstanding of the intended usage here. The way this should work is that the user decides:

 - What type of item they want to operate on (`source`)
 - Which items they want to operate on (`datetime`, `everything`, `ids`)
 - And what they want to do to those items (`update`, `delete`)

So, if I'm using this to update all recap content, I should say something like:

    --update --everything --type recap

And if I said that, it'd never get to this line, because it would get stopped on the line that reads `if options.get('everything')`. So, long story short, instead of putting this logic here, you should put it in the `add_or_update_all` method. This is an example of a different branch where I'm adding similar functionality for searching People:

```
def add_or_update_all(self):
        """
        Iterates over the entire corpus, adding it to the index. Can be run on
        an empty index or an existing one.

        If run on an existing index, existing items will be updated.
        """
        self.stdout.write("Adding or updating all items...\n")
        if self.type == Person:
            q = self.type.objects.filter(is_alias_of=None)
        else:
            q = self.type.objects.all()
        items = queryset_generator(q, chunksize=5000)
        count = q.count()
        self._chunk_queryset_into_tasks(items, count)
```